### PR TITLE
[docs] added missing actions in action reference

### PR DESCRIPTION
**Pull request by withshubh**
#### Changes
- Component specific action doc - Control Component
- Added missing docs - show modal, close modal, run query, set var, unset var etc
- updated organization to workspace in sidebar

### CODE REVIEW

File: docs/docs/actions/control-component.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (withshubh) [inline docs/docs/actions/control-component.md:105]:
```suggestion
```

**Comment by shubh22** [comment]
Task linked: [CU-2me529w CSA Documentaion](https://app.clickup.com/t/2me529w)

**Comment by withshubh** [comment]
@kavinvenkatachalam Please review the CSA doc (Control Component Action)

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@withshubh I guess you missed out `FilePicker` CSA in the `control-component.md` . Can you please check and add it?

File: docs/docs/actions/control-component.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+| Text Area | Set text, Clear |
+| Modal     | Show, Close |
+| Table   | Set page      |
+| Dropdown   | Select option |
+| Multiselect   | Select option, Deselect option, Clear selection |
+| Map   | Set location |
+| Checkbox   | Set checked |
+| Radio button | Select option |
+| Tabs   | Set tab |
+| Color picker   | Set color |
```
Comment (withshubh) [inline docs/docs/actions/control-component.md:28]:
```suggestion
| Color picker   | Set color |
| File picker    | Clear files |
```

**Comment by withshubh** [comment]
Added file picker CSA @kavinvenkatachalam

**Review by kavinvenkatachalam — APPROVED** [review]
LGTM

**Pull request merged**